admin/base/views.py

```diff
@@ -8,7 +8,10 @@
 
 @login_required
 def home(request):
-    context = {'user': request.user}
+    context = {
+        'user': request.user,
+        'user_groups': map(lambda g: g.name, request.user.groups.all())
```

Comment:
You should be able to do `request.user.groups.all().values_list('name', flat=True)` and it will do the same as your map but do it in sql.